config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id", to: "guest_rooms#create"
```

Comment:
Your controller organization is very RESTful. The only suggestion I have for the path is to include /rooms in there to indicate that the path is going to create an association between a given room and given guest, so it should include both resources. So it would instead be:
`post "/guests/:id/rooms", to: "guest_rooms#create"`